### DISCUSSION TITLE
Can't view the speed graph in the web UI

### DISCUSSION THREAD

**Discussion by purpledalek** [userdocs/qbittorrent-nox-static#151] [General]
I can't seem to find the option to view the upload/download speed graph in the web UI on qbittorrent-nox version 4.5.2 on Linux. I found a [forum post](https://forum.qbittorrent.org/viewtopic.php?t=4261) that said it apparently got added in version 3.3.4, however i can't seem to find it. i also found [this reddit post](https://www.reddit.com/r/qBittorrent/comments/kucmal/how_to_display_speed_graph_qbittorrent_v431_web_u/) showing a button in version 4.3.1 (top comment on the reddit post), however that button doesn't show in my webUI. Is there a setting i need to change to show the option?

**Comment by userdocs** [comment]
I don't think this is a webui setting. Most comments or replies you linked seem to reply based on the Desktop app, not the webui.

The code for this all seems to be under the `gui` and not the `webui`

https://github.com/search?q=repo%3Aqbittorrent%2FqBittorrent%20speed%20graph&type=code

If you want clarification ask the upstream project https://github.com/qbittorrent/qBittorrent/discussions

**Reply by purpledalek** [reply]
thank you, i have asked the upstream project
